core/frontend/src/components/app/PowerMenu.vue

```diff
@@ -185,8 +187,7 @@ export default Vue.extend({
           if (error.code === 'ECONNABORTED') {
             return
           }
-          const message = error.response?.data?.detail ?? error.message
-          notifications.pushError({ service: commander_service, type: 'SHUTDOWN_FAIL', message })
+          notifier.pushBackError('SHUTDOWN_FAIL', error)
```

Comment:
Couldn't 'BACKEND_OFFLINE' and etc be enumerations?

Reply:
That's indeed possible, but this is an error string for logging and human/machine readable.
Maintaining the enum may be problematic, share it between all services, having each service an enum, share and deal with multiple types for typescript and etc. There is no advantage doing such thing.